.gitmodules

```diff
@@ -134,5 +134,6 @@
 	path = tree-sitter-org
 	url = https://github.com/milisims/tree-sitter-org.git
 [submodule "tree-sitter-ziggy"]
+	# TODO switch back to kristoff-it when https://github.com/kristoff-it/ziggy/pull/29 is merged
 	path = tree-sitter-ziggy
-	url = https://github.com/kristoff-it/ziggy.git
+	url = https://github.com/MFAshby/ziggy
```

Comment:
We don't actually build ziggy anywhere here. There is no reason to change the ziggy upstream or the ziggy commit reference. Please remove it.

Reply:
ah thanks, removed the submodule in https://github.com/neurocyte/tree-sitter/pull/1/commits/da8fc99e847bb5f196c1c5ed8c3b6c3a7fe2f1a9